modules/InductionCalculator.py

```diff
@@ -76,7 +76,7 @@ def _loadConfig(self, **kwargs):
         '''
         # load info
         self._numCoilTurns              = kwargs.get('num_coil_turns', 4000)
-        self._usingAnalytic             = kwargs.get('using_analytic', True)
+        self._usingAnalytic             = kwargs.get('using_analytic', False)
```

Comment:
Why change the default from using analytic to not using?

Reply:
it doesn't matter(i just forget deleting that change from pull requests)